### PR TITLE
Make all the 3 WMs to work as similar as possible

### PR DESCRIPTION
**Pull request by Davoodeh**
This is an example related to the issue that I just opened about the divergence in shortcuts and behavior in different WMs (#780).
I stated there that i3 is kinda left behind and its shortcuts differ with DWM.

This PR aims to fix that and update everything to be as close as possible to DWM.

Spacings are deleted to make the files more similar to DWM config (and honestly they were extra).
Comments lines are deleted when they were unnecessary or non-existent in DWM config. On the other hand, regarding to that change, I expanded some flags to make them more descriptive without a comment.

`sysact` is more diverse and supports any WM/DE. (perhaps I should make another PR for it?)

### CODE REVIEW

**Comment by MYDavoodeh** [comment]
Pamixer is replaced with pulsemixer this far and all shortcuts are changed to fit dwm.

**Comment by MYDavoodeh** [comment]
I tried everything once and it seemed that everything just works fine.

Most of the changes are cleanups and sorts. Every config file, just like config.h, has keys in "gravemark to backslash order"

I tried to rearrange keys so system/wm-specific keys stay the same across both WMs. t thro o and g thro semicolon, z, x, s have WM functions.

Guess the next step, now, is to document them.

**Comment by vladdoster** [comment]
I don't think there is a push to support all 3 wm's. 

Luke even [removed i3 from dependencies](https://github.com/LukeSmithxyz/LARBS/commit/5ffc31bbc1b063c80a43c60affd8ef2b39e1d14a). This just seems like it adds a lot of possible headaches for a small percentage of users, if any.

**Comment by MYDavoodeh** [comment]
Yeah there is no push but I am able to do it so why not.

**Comment by et-8** [comment]
Regarding adding code to `.local/bin/sysact` and maybe other files.
I think it would be a good idea to move the files that are no longer actively used by LARBS to a separate repo like "Old Voidrice".

Adding code for WM´s not used by the current version of LARBS to files actively used by  the current version of LARBS will make those files and LARBS more difficult to debug.
#780

**Comment by vladdoster** [comment]
@et-8

 how does it make things harder to debug? I don't think that is true since you'd have no reason to look through those files.

If moved to a new repo, that would be more confusing and Make new features added to voidrice be added to a 2nd place. This sounds like a bad idea. 

@MYDavoodeh 

Because it makes things more complex for things that aren't currently supported? Just because you can doesn't mean you should...

**Comment by MYDavoodeh** [comment]
Thanks for commenting @et-8
I also agree with @vladdoster.
As you can see that issue is closed. The conclusion was: there is basically no reason to do so.

I mean come on why you take it so seriously it's just 3 extra lines xD

**Comment by vladdoster** [comment]
@MYdavoodeh 

The changes are not minimal. You change 7 files and touch 600 lines.

There is no support for bspwm so I don't understand why youre trying to add changes for it.

In i3 config:
Why are the lines spacing keybindings deleted? Commits should be atomic. 
Why are commented lines deleted? They are there to give users an idea of what other keys can be bound? Regardless, they don't effect any changes you made, so why delete them?

**Comment by MYDavoodeh** [comment]
@vladdoster thank you sincerly for your questions...

First I add my opinion about the lines you added then I try to answer them if I can.
That's true about the Unix philosophy, but sysact is "A dmenu wrapper script for system functions." According to that I see it doesn't do the very one thing that it supposes to do. It's simply incomplete.

Changes to that one file are minimal, perhaps I should have mentioned it (edited the original comment).

Questions:
Spacings are deleted to look more similar to DWM config (and honestly they were extra).
Comments lines are deleted when they were unnecessary or non-existent in DWM config. On the other hand, regarding to that change, I expanded some flags to make them more descriptive without a comment.
Most comments are descriptive on their own.

**Comment by vladdoster** [comment]
@MYDavoodeh 

## Sysact
Sysact does one thing.... it performs system actions and nothing else. It wouldn't make sense to have a script for reboot, poweroff, logout. 

## i3
In general, you do not want to mess with formatting in commits in someone else's repository. You only want to commit code. It also makes the diff longer for no reason. I've been scolded for trying to do this in other projects. 

If youre going to fix formatting in i3 config, there are still lines that use tabs instead of spaces. You should fix those. I left a few comments to show you lines that need to be fixed.

Since i3 isnt being used by Luke, I think that changing things at this point is futile. It would also might confuse/break things for users of the current i3 config if they try to merge any future voidrice changes.

**Comment by MYDavoodeh** [comment]
Thank you for your advice...

If that someone else don't agree with new changes they reject it... what's the worst that can happen? :P

The tab issues is not only limited to that file. It's all over the place. Mixed indents are everywhere in this repo. I will make a separate PR for that.

I'm a current i3 user. It doesn't break anything. If I opened this I certainly tested it :)
Also a few of my friends are i3 users. They demanded the changes that's why I opened this. Since when making changes at files is futile? There are broken shortcuts in it. I don't understand the argument at this point xD

**Comment by vladdoster** [comment]
@MYDavoodeh 

The author of the PR should not include major style changes combined with other changes. It makes it hard to see what is being changed in the PR, makes merges and rollbacks more complex, and causes other problems. For example, if the author wants to reformat the whole file, have them send you just the reformatting as one PR, and then send another PR with their functional changes after that.

**Comment by MYDavoodeh** [comment]
I totally understand and appreciate it.

So, I tried to make the reformattings one in one different commit so it will be easiser to check. But guess I just have to do everything in another PR.

Thanks again for everything!

**Comment by vladdoster** [comment]
Can you close this?

File: .config/i3/config [hunk 1]
```diff
@@ -17,343 +17,230 @@ set $reboot sudo -A reboot
...
 bindsym $mod+k			focus up
 bindsym $mod+Shift+k		move up 30
 bindsym $mod+Ctrl+k		move workspace to output up
-
 bindsym $mod+l			focus right
 bindsym $mod+Shift+l		move right 30
 bindsym $mod+Ctrl+l		move workspace to output right
+bindsym $mod+semicolon		workspace next
+bindsym $mod+Shift+semicolon	move container to workspace next
+bindsym $mod+apostrophe		exec --no-startup-id ddspawn dropdowncalc -f mono:pixelsize=24
```
[the remark this answers is not in the document]
Comment (MYDavoodeh) [reply .config/i3/config:155]: Maybe I'm mistaken but I think this is all tabs already

File: .config/i3/config [hunk 2]
```diff
@@ -17,343 +17,230 @@ set $reboot sudo -A reboot
...
-bindsym $mod+Shift+x		exec --no-startup-id prompt "Shutdown computer?" "$shutdown"
-
-bindsym $mod+c			exec --no-startup-id cabl
-bindsym $mod+Shift+c		exec --no-startup-id camtoggle
-
-bindsym $mod+v			exec --no-startup-id $term -e $EDITOR -c "VimwikiIndex"
-#bindsym $mod+Shift+v
-
+bindsym $mod+Shift+z		gaps inner current plus 5
+bindsym $mod+x			gaps outer current minus 5
```
[the remark this answers is not in the document]
Comment (MYDavoodeh) [reply .config/i3/config:162]: Maybe I'm mistaken but I think this is all tabs already

**Pull request closed without merging**